testsuite/integration-arquillian/tests/base/src/test/resources/META-INF/keycloak-server.json

```diff
@@ -273,5 +273,12 @@
 
     "saml-artifact-resolver": {
         "provider": "${keycloak.saml-artifact-resolver.provider:default}"
+    },
+
+    "security-profile": {
```

Comment:
I think same changes can be good also in `testsuite/utils/src/main/resources/META-INF/keycloak-server.json` , which is used by `KeycloakServer` utility.

Reply:
Done! I don't really know if it's needed because the quarkus server uses the startup options but it does not make any harm.